src/index.js

```diff
@@ -1,18 +1,21 @@
+import * as Cesium from "../node_modules/cesium"
```

Comment:
The webpack file has aliased the Cesium Source folder as `cesium`, so this could be 

```suggestion
import * as Cesium from "cesium"
```

It's pretty common to use aliases, or at least have an alias for `node_modules` to avoid relative paths.

If you don't want to use the source files directly, you can configure the alias to point at the top level cesium path as you use here.

If you chose not to use the alias, it should be removed from the config file altogether.